book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -208,159 +197,54 @@ Then we call `.run()` on `game` which starts the game loop. The game will
+let game_data = GameDataBuilder::default()
+    .with_bundle(
+        RenderingBundle::<DefaultBackend>::new()
+            // The RenderToWindow plugin provides all the scaffolding for opening a window and drawing on it
+            .with_plugin(
+                RenderToWindow::from_config_path(display_config_path)
+                    .with_clear([0.0, 0.0, 0.0, 1.0]),
+            )
+            // RenderFlat2D plugin is used to render entities with `SpriteRender` component.
+            .with_plugin(RenderFlat2D::default()),
+    )?;
+# Ok(()) }
 ```
 
-Here we are creating an `ExampleGraph` struct and implementing a `GraphCreator` trait for it.
-This trait is responsible for setting up all the details of our rendering pipeline.
-
-> **Note:** This setup code is directly using `rendy` crate to define the rendering.
-> You can read about its concepts in the [rendy graph docs][graph].
-
-The important thing to note is that this renders a black background.
-It is also ready to draw 2D sprites for us, which we will use in the next chapter.
+Here we are adding a `RenderingBundle`. Bundles are essentially sets of systems
+preconfigured to work together, so you don't have to write them all down one by one.
 
-If you want to use a different background color, you can tweak the RGBA
-values inside `ClearValue::Color`. Values range from `0.0` to `1.0`,
-so to get that cool green color you can try `[0.00196, 0.23726, 0.21765, 1.0]`.
-
-Now let's pack everything up and run it back in the `main()` function. We have to
-expand the existing `GameDataBuilder` with `RenderingSystem` that uses our graph:
+> **Note:** We will cover systems and bundles in more details later, for now, think of the
+> bundle as a group of functionality that together provides a certain feature to the engine.
+> You will surely be writing your own bundles for your own game's features soon.
 
-```rust,ignore
-let game_data = GameDataBuilder::default()
-    // The WindowBundle provides all the scaffolding for opening a window
-    .with_bundle(WindowBundle::from_config_path(display_config_path))?
-    // A Processor system is added to handle loading spritesheets.
-    .with(
-        Processor::<SpriteSheet>::new(),
-        "sprite_sheet_processor",
-        &[],
-    )
-    // The renderer must be executed on the same thread consecutively, so we initialize it as thread_local
-    // which will always execute on the main thread.
-    .with_thread_local(RenderingSystem::<DefaultBackend, _>::new(
-        ExampleGraph::default(),
-    ));
-
-let assets_dir = app_root.join("assets/");
+The `RenderingBundle` is actually a bit special, as it doesn't really do anything on its own,
+but relies on it's own plugin system to define what should be rendered and how. We have to use
+`with_plugin` method to tell it that we want `RenderToWindow` and `RenderFlat2D` plugins.
+Those plugins will equip our renderer with ability to open a window and draw 2D sprites on it. 
```

Comment:
The `RenderingBundle` has a difference to most other bundles: It doesn't really do anything by itself. Instead, it relies on its own plugin system to define what should be rendered and how. We use the `with_plugin` method to tell it that we want to add the `RenderToWindow` and `RenderFlat2D` plugins. Those plugins will equip our renderer with the ability to open a window and draw sprites to it.